livechat/overview/get_started.rst

```diff
@@ -0,0 +1,83 @@
+==========================
+Get Started with Live Chat
+==========================
+
+Live chat is a cost-effective tool and a great way to bring visitors back to your website. It allows
+fast responses, human interaction, sales' improvement, accessible and convenient support, and even
+the reduce of services costs. Customers want to talk to you, so let's make it easy.
+
+Activate and set it up
+======================
+
+Once *Live Chat* is installed on your database, if your website was created with Odoo, the
+application is automatically added to it. All that is left to do is to go to
+:menuselection:`Website --> Configuration --> Settings --> Live Chat`.
+
+.. image:: media/livechat_settings.png
+   :align: center
+   :height: 300
```

Comment:
Weirdly, the `height ` command makes the image clickable in html. the link takes the user out of the doc page, to the image alone.
I'm just pointing it to you so you know that it happens, but that's it.